src/components/TokenGeneralInfo.js

```diff
@@ -210,7 +181,7 @@ class TokenGeneralInfo extends React.Component {
       return (
         <p className="mt-2 mb-4">
           <strong>{t`Always show this token:`}</strong> {
-          this.state.alwaysShow
```

Comment:
Yes it is, I guess that since the `renderAlwaysShowTokenCheckbox` is not memoized, react will run through it and since it is returning a different value, it will re-render with the new value